assets/js/utils/rtcUtils.ts

```diff
@@ -37,31 +37,51 @@ export const connectWebrtc = async (
     console.error("Couldn't get camera permission:", error);
   }
 
+  const updateStream = (toAdd: boolean, stream: MediaStream | null, name: string) => {
+    // adds new stream when toAdd is true, else removes the stream
+    if (toAdd && stream != null) presenterStreams[name] = stream;
+    let new_elem = {};
+    new_elem[name] = true;
+    setStreamsAvailable({ ...streamsAvailable, ...new_elem });
+  };
```

Comment:
Why are we doing this in one function? What is the problem with creating `addStream` and `removeStream` functions?